core/fslock/fslock.cpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "fslock/fslock.hpp"
+
+#include <boost/filesystem/fstream.hpp>
+#include <boost/filesystem/operations.hpp>
+#include "fslock/fslock_error.hpp"
+
+namespace fc::fslock {
+  // TODO(artyom-yurin): Should be unlocked if process died
```

Comment:
there should be a task created for every TODO

Reply:
Added task [FIL-115]